org.adempiere.base/src/org/compiere/model/X_AD_Package_Exp_Detail.java

```diff
@@ -950,6 +968,10 @@ public String getTarget_Directory ()
 	public static final String TYPE_Reference = "REF";
 	/** Role = S */
 	public static final String TYPE_Role = "S";
+	/** Shell Script = SH */
+	public static final String TYPE_ShellScript = "SH";
+	/** Code Snippet = SNI */
+	public static final String TYPE_CodeSnippet = "SNI";
```

Comment:
It seems this and TYPE_File_CodeOrOther is not use anywhere ?

Reply:
OK, please check https://github.com/idempiere/idempiere/pull/898